solidity/contracts/BancorNetworkPathFinder.sol

```diff
@@ -38,7 +39,7 @@ contract BancorNetworkPathFinder is ContractRegistryClient {
       * 
       * @return a path from the source token to the target token
     */
-    function generatePath(address _sourceToken, address _targetToken) public view returns (address[] memory) {
+    function findPath(address _sourceToken, address _targetToken) public view returns (address[] memory) {

```

Comment:
@yudilevi : We can technically keep this interface as is, and just connect the already deployed path-finder to the (new) network. Any special reason which justifies this API-change?

Reply:
yeah, it's part of the property style API and next time we upgrade the pathfinder, I don't want to break the network, so might as well do now, anyone who uses the current pathfinder can still keep using it, so no harm done here.